mkosi/resources/mkosi.md

```diff
@@ -395,8 +395,11 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
 
 : The distribution to install in the image. Takes one of the following
   arguments: `fedora`, `debian`, `ubuntu`, `arch`, `opensuse`, `mageia`,
-  `centos`, `rhel-ubi`, `openmandriva`, `rocky`, `alma`. If not
-  specified, defaults to the distribution of the host.
+  `centos`, `rhel-ubi`, `openmandriva`, `rocky`, `alma`, `none`.
```

Comment:
Can you update the `Supported Distributions` section as well?

Reply:
Haven't seen that section before. I removed the explanation from --distribution down to that section and expanded it a little bit.